packages/legacy/core/App/components/views/KeyboardView.tsx

```diff
@@ -1,10 +1,11 @@
 import React from 'react'
 import { KeyboardAvoidingView, Platform, ScrollView } from 'react-native'
 import { SafeAreaView } from 'react-native-safe-area-context'
+import { ColorPallet } from '../../theme'
```

Comment:
Theme items should come from `useTheme` so that themes from wallets that extend Bifold are used rather than just the base Bifold theme